.pre-commit-config.yaml

```
@@ -0,0 +1,15 @@
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v2.4.0
+    hooks:
+    -   id: trailing-whitespace
+    -   id: end-of-file-fixer
+    -   id: check-yaml
+    -   id: check-added-large-files
+
+repos:
+  - repo: https://github.com/psf/black
+    rev: 19.10b0 # Replace by any tag/version: https://github.com/psf/black/tags
+    hooks:
+      - id: black
+        language_version: python3 # Should be a command that runs python3.6+
```

Does `repos` key need to be repeated? I believe this way (:point_down:) works as well. 

Also, is there a reason for us to use `v2.4.0` and not `v3.2.0`? I believe a lot of projects that use `v.2.4.0` is due to the fact that from `v3.0.0` and on, `pre-commit-hooks` requires `Python >= 3.6.1` (and a lot of systems still have `Python 3.5` as a default)... considering this is not a limitation for us, we could make use a more recent version without problems! :grimacing: 

```suggestion
repos:
  - repo: https://github.com/pre-commit/pre-commit-hooks
    rev: v3.2.0
    hooks:
    -   id: trailing-whitespace
    -   id: end-of-file-fixer
    -   id: check-yaml
    -   id: check-added-large-files

  - repo: https://github.com/psf/black
    rev: 19.10b0 # Replace by any tag/version: https://github.com/psf/black/tags
    hooks:
    -   id: black
        language_version: python3 # Should be a command that runs python3.6+
```

---

Nice, will change it.